website/src/middlewares/apiThrottle.js

```diff
@@ -0,0 +1,22 @@
+var nconf = require('nconf');
+var limiter = require('connect-ratelimit');
+
+var IS_PROD = nconf.get('NODE_ENV') === 'production';
+
+module.exports = function(app) {
+  if (!IS_PROD) return;
+  app.use(limiter({
+    end:false,
+    catagories:{
```

Comment:
It was on their side but has been fixed in 0.0.7 and we use that https://github.com/dharmafly/connect-ratelimit/commit/e698edca50aaa3b6aecd81181648ae16f34e8d58